src/Common/src/CoreLib/System/Globalization/DateTimeFormatInfo.cs

```diff
@@ -2442,7 +2343,136 @@ internal TokenHashValue[] CreateTokenHashTable()
             return (temp);
         }
 
-        private void AddMonthNames(TokenHashValue[] temp, ReadOnlySpan<char> monthPostfix = default)
+        private void PopulateSpecialTokenHashTable(TokenHashValue[] temp, ref bool useDateSepAsIgnorableSymbol)
```

Comment:
This has been moved into it's own method because it's quite large and we want the linker to remove it.